R/model-interface.R

```diff
@@ -77,6 +77,36 @@ brms_model <- function(formula, family, ...) {
   )
 }
 
+add_prediction_interval <- function(model, data, alpha) {
+  UseMethod("add_prediction_interval")
+}
```

Comment:
So, I am looking at this first thing after waking up, with a coffee, and my thought before opening the PR was: did he create a generic for the PI so we can expand on other models?

\m/ ^ ^ \m/